.github/workflows/test.yaml

```
@@ -34,10 +34,10 @@ jobs:
 
     steps:
       - uses: actions/checkout@v4
-      - name: Setup Go 1.24.0
+      - name: Setup Go
         uses: actions/setup-go@v4
```

actionlint (1.7.7)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)